msal/src/request/java/com/microsoft/identity/client/BaseRequest.java

```diff
@@ -204,7 +204,7 @@ AuthenticationResult postTokenRequest() throws MsalUiRequiredException, MsalServ
 
         final AccessTokenCacheItem accessTokenCacheItem;
         // Switch to toggle using common cache (or not)
-        boolean useCommonCache = true;
+        boolean useCommonCache = false;
```

Comment:
why have you turned this off?

Reply:
This is by design. Sync'ed with Brian

Reply:
This change is from @iambmelt

Reply:
@iambmelt The unit tests for common-core cache is not fixed currently and here it only switched off the useCommonCache flag, right?

Reply:
@heidijinxujia @addev-ashish I've switched off these tests because this branch uses the _old_ 'new' cache that Shane and I developed back in November/December of last year (before the new cache schema was authored). The branch `iambmelt/account-credential-schema` has the _new_ 'new' cache which has the schema updates and a different API.